src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -553,7 +709,25 @@ class ExplorerButtonGroup extends React.Component {
     } else if (buttonConfig.type === 'manifest' && this.state.manifestEntryCount > 0) {
       buttonTitle = `${buttonConfig.title} (${humanizeNumber(this.state.manifestEntryCount)})`;
     }
-    const btnTooltipText = (this.props.isLocked) ? 'You only have access to summary data' : buttonConfig.tooltipText;
+
+    let tooltipEnabled = false;
+    let btnTooltipText = '';
```

Comment:
nitpicking: seems you can set the default value for these 2 the same from L728-729 and get rid of the `else {}`